lib/stripe/entity.ex

```diff
@@ -0,0 +1,55 @@
+defmodule Stripe.Entity do
+  @callback __from_json__(data :: map) :: map
```

Comment:
This one will definitely require documentation.

We want to discourage library users from using it directly, but we also want to explain it to future contributors, so they can use it correctly when implementing new features.